src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -94,6 +95,14 @@ public TimeSpan UserProcessorTime
             }
         }
 
+        public int HandleCount
+        {
+            get
+            {
+                return Directory.GetDirectories("/dev/fd", "*", SearchOption.TopDirectoryOnly).Length;
```

Comment:
Yeah. OSX doesn't have a `/proc/pid/fd` equivalent. 

solutions:
- Return the FDs of the current process (current solution)
- Return 0 (previous solution)
- Patch together something with lsof